lib/provider.js

```diff
@@ -64,6 +65,9 @@ module.exports = {
         console.log('');
       }
 
+//Fix for windows as BufferedNodeProcess doesn't spawn properly; see Atom issue 2887.
+//May not be the best way but works for now.
+      if (process.platform !== 'win32') {
```

Comment:
Also, I'm unsure of the code style used in this project, but it would likely read easier if the comment was aligned with the `if` statement just below it.